From f422544118cbdfc3bba7b7c5189e18147acb9047 Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.i.king@gmail.com>
Date: Tue, 28 Nov 2023 09:53:04 +0000
Subject: [PATCH] net: mana: Fix spelling mistake "enforecement" ->
 "enforcement"

There is a spelling mistake in struct field hc_tx_err_sqpdid_enforecement.
Fix it.

Signed-off-by: Colin Ian King <colin.i.king@gmail.com>
Signed-off-by: Shradha Gupta <shradhagupta@linux.microsoft.com>
Link: https://lore.kernel.org/r/20231128095304.515492-1-colin.i.king@gmail.com
Signed-off-by: Jakub Kicinski <kuba@kernel.org>
---
 drivers/net/ethernet/microsoft/mana/mana_en.c      | 2 +-
 drivers/net/ethernet/microsoft/mana/mana_ethtool.c | 4 ++--
 include/net/mana/mana.h                            | 2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

Index: linux/drivers/net/ethernet/microsoft/mana/mana_en.c
===================================================================
--- linux.orig/drivers/net/ethernet/microsoft/mana/mana_en.c
+++ linux/drivers/net/ethernet/microsoft/mana/mana_en.c
@@ -2489,7 +2489,7 @@ void mana_query_gf_stats(struct mana_por
 	apc->eth_stats.hc_tx_err_eth_type_enforcement =
 					     resp.tx_err_ethtype_enforcement;
 	apc->eth_stats.hc_tx_err_sa_enforcement = resp.tx_err_SA_enforcement;
-	apc->eth_stats.hc_tx_err_sqpdid_enforecement =
+	apc->eth_stats.hc_tx_err_sqpdid_enforcement =
 					     resp.tx_err_SQPDID_enforcement;
 	apc->eth_stats.hc_tx_err_cqpdid_enforcement =
 					     resp.tx_err_CQPDID_enforcement;
Index: linux/drivers/net/ethernet/microsoft/mana/mana_ethtool.c
===================================================================
--- linux.orig/drivers/net/ethernet/microsoft/mana/mana_ethtool.c
+++ linux/drivers/net/ethernet/microsoft/mana/mana_ethtool.c
@@ -43,8 +43,8 @@ static const struct {
 	 offsetof(struct mana_ethtool_stats, hc_tx_err_eth_type_enforcement)},
 	{"hc_tx_err_sa_enforcement", offsetof(struct mana_ethtool_stats,
 					      hc_tx_err_sa_enforcement)},
-	{"hc_tx_err_sqpdid_enforecement",
-	 offsetof(struct mana_ethtool_stats, hc_tx_err_sqpdid_enforecement)},
+	{"hc_tx_err_sqpdid_enforcement",
+	 offsetof(struct mana_ethtool_stats, hc_tx_err_sqpdid_enforcement)},
 	{"hc_tx_err_cqpdid_enforcement",
 	 offsetof(struct mana_ethtool_stats, hc_tx_err_cqpdid_enforcement)},
 	{"hc_tx_err_mtu_violation", offsetof(struct mana_ethtool_stats,
Index: linux/include/net/mana/mana.h
===================================================================
--- linux.orig/include/net/mana/mana.h
+++ linux/include/net/mana/mana.h
@@ -369,7 +369,7 @@ struct mana_ethtool_stats {
 	u64 hc_tx_err_vlan_enforcement;
 	u64 hc_tx_err_eth_type_enforcement;
 	u64 hc_tx_err_sa_enforcement;
-	u64 hc_tx_err_sqpdid_enforecement;
+	u64 hc_tx_err_sqpdid_enforcement;
 	u64 hc_tx_err_cqpdid_enforcement;
 	u64 hc_tx_err_mtu_violation;
 	u64 hc_tx_err_inval_oob;
